packages/liveblocks-node/src/index.ts

```diff
@@ -90,8 +95,20 @@ export async function authorize(
   } catch (er) {
     return {
       status: 403,
-      body: 'Call to "https://liveblocks.io/api/authorize" failed. See "error" for more information.',
+      body: 'Call to "https://api.liveblocks.io/v2/rooms/{roomId}/authorize" failed. See "error" for more information.',
```

Comment:
Did you mean to expand this `{roomId}` position here in the error message? Or is this intended to be a template placeholder there? From other systems, I'm used to the `/v2/rooms/:roomId` syntax for those REST/URL placeholders, but I guess that's just an arbitrary choice. As long as we're consistent about a single syntax, I'm good with this 👍